src/privateai_client/components/request_objects.py

```diff
@@ -512,6 +512,41 @@ def fromdict(cls, values: dict):
             )
 
 
+class OCROptions(BaseRequestObject):
+    default_ocr_system = "azure_computer_vision"
```

Comment:
IIRC, this is `paddleocr` in deid 🤔 

Do we need to set the default here though? It'll be handled by the pydantic in deid.

Reply:
Thanks Girish! in this client, it sets default value separately. Are we okay to swap to `paddleocr` here?

e.g. 
`class ProcessedText(ProcessedMarkerText, ProcessedMaskText, ProcessedSyntheticText):
    default_type = "MARKER"
    valid_types = ["MARKER", "MASK", "SYNTHETIC"]`

Reply:
Yes, it is the defeault OCR. Reference: https://github.com/privateai/deid/blob/acb817f62fe1b0e2dfee9d9b34fe9534713fdb95/app/models/request.py#L619

@a-guiducci which docs? docs.private-ai.com?

IMO defaulting to paddle make sense, as we ship it with the container and user can start with with with least friction.

@yuya-kitani should we bring this up in dev channel/meeting?

Reply:
Ya definitely agree that paddle makes more sense